leaderboard/llvm_codesize/e_greedy/README.md

```diff
@@ -0,0 +1,49 @@
+# Epsilon-Greedy Search
+
+**tldr;**
+A greedy search policy that at each step will choose to select an action
```

Comment:
This doesn't seem to mention your `greedy` policy, especially that you are only rolling out the trajectory by **one** step and picking the best.

Reply:
Thanks, good spot! Fixed